### DISCUSSION TITLE
Please add version number to zip filename

### DISCUSSION THREAD

**Discussion by vidur501** [bnpr/Malt#264] [Feedback & Feature Requests]
Hello,

every version that I download is named BlenderMalt-Linux.zip, no version number, no indication if it is dev or release.
Additionally, there is no version info displayed in Blender for the Release version.

When trying to report which version is working and which is not, I'm lost and have to re-download the versions from github and name them myself.

So please add version number and related info to the name of your zips.

**Comment by pragma37** [comment]
Once I release the stable 1.0 version I plan to follow semantic versioning.  
I'll take this request into account too. 👍
